Updated tooltips in App.js

tuankhoin opened the pull request:
Added definition tooltips. You can now hover the mouse at keywords for definitions.
For documentation: [https://material-ui.com/components/tooltips/#arrow-tooltips]()
Contact: tuankhoin@student.unimelb.edu.au

freddiewanah commented:
Hey, I'd like to review your PR, but it seems that there're some conflicts that need to be fix. And since you're introducing a new lib to the site, can you send some screenshot of the tooltip you've build and make sure that it doesn't influence our current component?

tuankhoin commented:
Yes, I'll just screen cap and highlight what I've added to the file for you. I've noticed that there was some changes that I didn't make, probably because I was modifying on an older version of the repo.
The libraries that I've added, 4 in total:
![image](https://user-images.githubusercontent.com/58412534/77321527-6d88e000-6d66-11ea-87e8-a526bce84dcc.png)
Added HtmlToolTip:
![image](https://user-images.githubusercontent.com/58412534/77321753-c22c5b00-6d66-11ea-811b-6117db549e76.png)
Modified the stat function's return by adding tooltip inside each tag:
![image](https://user-images.githubusercontent.com/58412534/77321861-f0aa3600-6d66-11ea-8190-001cc745cdb5.png)
A demo of what I implemented:
![2020-03-24](https://user-images.githubusercontent.com/58412534/77321982-1e8f7a80-6d67-11ea-95cf-341b9ba9d212.png)

tuankhoin commented:
Hi, I've checked the conflict. Seems like it is not related to my modification, so I guess that it may have occurred because someone modified the origin while I was adding the code in. I have screenshot my modifications above for you. If you need anything, feel free to contact me. Have a good night Han!

gamma032steam commented:
Resolved the conflict.

gamma032steam commented on src/App.js:
```diff
@@ -387,14 +619,32 @@ function Stat({
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Deaths</Typography>
+              <em>{"All cases that resulted in death, mostly found in the elderly."}</em>
```
This one is probably unnecessary.

tuankhoin replied:
It's just there to make sure that any tag will have a definition. I find it awkward to have only one with a definition when hovered. But you guys can remove it if you want to. Just a personal pov.

gamma032steam commented on src/App.js:
```diff
@@ -387,14 +619,32 @@ function Stat({
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Recovered</Typography>
+              <em>{"All cases that are no longer infected."}</em>
```
Good place to elaborate on what has to happen for a patient to be deemed 'recovered'.

tuankhoin replied:
Tbh I'm not actually sure about its definition as well haha. What do you think it could be?

tuankhoin commented:
Guys. Since I've unknowingly commiting to a new PR, and that the previous code that I was working on is an old version, I will now move to a new PR. You can head to PR #98 for further tooltips development.
